### DISCUSSION TITLE
Download problem

### DISCUSSION THREAD

**Discussion by sympan74** [aclap-dev/video-downloadhelper#1335] [Help and Q&A]
### Your question

hi
few days ago create problem with download from one site. previus days working normally, now in download files see just one color an in top of screen some pixels 
![image](https://github.com/user-attachments/assets/502771a9-ccf2-4fb1-857d-bdbba37481bc)
can help me with this ?

### Addon & CoApp version and details

version: 9.1.0.44
target: google
channel: stable
lang: el
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"8a1b3b63a57541c8"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36

**Comment by Mattmjs** [comment]
Unfortunately the video is protected with DRM, nothing can be done on it. The software doesn't circumvent it.

**Reply by Joh3nny** [reply]
Hi. I have the same problem. Two days ago the software worked fine but now the screen is full of pixels. It is strange both web sites protected the videos same time. May be is something that is not working fine now for any update. Could you check it?

**Reply by Joh3nny** [reply]
I just checked that I can record the screen, so I think is not a DRM problem

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

There are quite a few other discussions in this forum dealing with DRM.  You need to search this forum with `drm` as your search key.  Read some of the other discussions that your search gives you.  You will find out how to determine whether a particular video is protected by DRM.

Web sites change the way they operate all the time.  One day you can download from them, the next day they've protected all their content with DRM.  That's life.

If you really want help with a particular video, you have to post the URL of the web page on which it resides.
